.circleci/config.yml

```diff
@@ -123,6 +123,7 @@ jobs:
 
   install_test_node_22:
     executor: node_22
+    resource_class: large
```

Comment:
I guess you can remove this as it is now clarified that it is not related to nodejs version.